openedx_lti_tool_plugin/signals.py

```diff
@@ -38,21 +38,21 @@ def restrict_user_email_address(
     return
 
 
-@receiver(post_save, sender=LtiTool, dispatch_uid='create_access_configuration_on_lti_tool_creation')
-def create_course_access_configuration(
+@receiver(post_save, sender=LtiTool, dispatch_uid='create_configuration_on_lti_tool_creation')
```

Comment:
```suggestion
@receiver(post_save, sender=LtiTool, dispatch_uid='f{app_config.name}.create_lti_tool_configuration')
```
@Serafin-dev What do you think about add the app name to the dispatch_uid argument?